### PR TITLE
Bugfix/objects 1007

### PR DESCRIPTION
**Pull request by andreassiegel**
### What changes were proposed in this pull request?
- removes `try {} catch ()` blocks for `IllegalArgumentException` because they never really worked, and the exceptions are now handled in Framework's `RequestExceptionHandler` controller advice
- fixes the URN scheme, so that `bogus_<valid URN>` or `<valid URN>_bogus` is not valid anymore
- adds Jenkins build status 😎 
### How is this patch documented?

Code
### How was this patch tested?

existing tests and manually in Postman
#### Depends On

`RequestExceptionHandler` controller advice in https://github.com/SMARTRACTECHNOLOGY/smartcosmos-framework/pull/198

### CODE REVIEW

File: src/main/java/net/smartcosmos/dao/things/util/UuidUtil.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
     public static UUID getUuidFromUrn(String urn) throws IllegalArgumentException {
 
-        String urnScheme = "urn:.*:uuid:([A-F0-9]{8}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{12})";
+        String urnScheme = "^urn:.*:uuid:([A-F0-9]{8}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{12})$";
```
Comment (ghost) [inline src/main/java/net/smartcosmos/dao/things/util/UuidUtil.java:22]: What's this?  This is not related to the `IllegalArgumentException` work is it?

**Review by ghost — APPROVED** [review]

File: src/main/java/net/smartcosmos/dao/things/util/UuidUtil.java (hunk 1 above)
Comment (andreassiegel) [reply src/main/java/net/smartcosmos/dao/things/util/UuidUtil.java:22]: Well, you could consider it part of it:

Previously, URNs like `urn:thing:uuid:926a94f9-ab5c-448c-9109-c3d7dff2cf87NotValid` or  `This-is-not-a-valid-URN:urn:thing:uuid:926a94f9-ab5c-448c-9109-c3d7dff2cf87` did not throw any exception, because such values were considered valid URNs, but they actually are not.

Comment (ghost) [reply src/main/java/net/smartcosmos/dao/things/util/UuidUtil.java:22]: ok, that makes sense.  Are there tests?

Comment (andreassiegel) [reply src/main/java/net/smartcosmos/dao/things/util/UuidUtil.java:22]: Not yet. I'll add some.
